src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -42,16 +42,16 @@ def is_folder(path: str) -> bool:
 
 
 def validate_io_paths(
-    source: Sequence[str], output: Sequence[str], register: bool
+    source: Sequence[str], output: Sequence[str], *, for_registration: bool
 ) -> None:
     if len(source) == 0 or len(output) == 0:
         raise ValueError("Source/Output list must not be empty.")
 
-    if register:
+    if for_registration:
         if any(o.startswith("tiledb://") for o in output):
             raise ValueError(
-                "Invalid value of argument 'register'. \
-                             TileDB URIs require register argument to be set to False"
+                "Invalid value of argument 'register'"
+                "TileDB URIs require register argument to be set to False"
```

Comment:
@ktsitsi since this check is to catch tiledb URIs, let's say that in the exception message.

```suggestion
                "Output sequence contains a tiledb URI and this cannot be re-registered."
```